test/core/TakerOperations.t.sol

```diff
@@ -330,18 +314,17 @@ contract TakerOperationsTest is MangroveTest {
 
     expectFrom($(mgv));
     emit OfferFail($(base), $(quote), ofr, $(this), 1 ether, 1 ether, "mgv/makerTransferFail");
-    (uint successes, uint takerGot, uint takerGave,,) =
-      testMgv.snipesInTest($(base), $(quote), wrap_dynamic([ofr, uint(Tick.unwrap(offerTick)), 1 ether, 100_000]), true);
+    vm.expectEmit(true, false, false, false, $(mgv));
+    emit Credit($(mkr), 0); // credited amount should be "mkr_provision - penalty"; Not including this in the emit assertion, as penalty is difficult to maintain.
```

Comment:
Same as above!

Reply:
Same answer/question as above :-)

Reply:
Addressed in [a21d89d](https://github.com/mangrovedao/mangrove-core/pull/454/commits/a21d89dbc0f556eda8cdea09e69e21f89da9b907)